src/_pytest/_code/code.py

```diff
@@ -217,8 +217,8 @@ def path(self) -> Union[py.path.local, str]:
         """ path to the source code """
         return self.frame.code.path
 
-    @property
-    def locals(self) -> Dict[str, Any]:
+    @property  # noqa : A003
```

Comment:
This doesn't seem correct right? We are not overriding `property`.

Reply:
resolving as per my comment below your review. Please reopen if you disagree